resolvetosavelives/healthicons
# Modal with routes

Adds support for routing for the modals

### To Test
 - load the page
 - open an icon modal (path should change, page should not scroll to the top)
 - close the modal (path should change, page scroll position should be maintained)
 - try searching and/or filtering by icon style
 - open/close modal and the search settings should be maintained
